assets/src/edit-story/components/library/panes/pageTemplates/pageTemplate.js

```diff
@@ -122,6 +122,9 @@ function PageTemplate(
       pageSize={pageSize}
       role="listitem"
       ref={ref}
+      // TODO: Investigate
+      // See https://github.com/google/web-stories-wp/issues/6671
+      // eslint-disable-next-line styled-components-a11y/no-noninteractive-tabindex
       tabIndex={0}
```

Comment:
That's for being able to use arrow keys for navigating through the templates, there's another `button` inside it, otherwise, it could be a button instead. Perhaps we could try using a button `role` for it. Needs further investigation.